resources/product-listing.js

```diff
@@ -19,7 +19,38 @@ function ProductListing(shopify) {
   this.key = 'product_listing';
 }
 
-assign(ProductListing.prototype, omit(base, ['create', 'update']));
+assign(ProductListing.prototype, omit(base, ['create', 'update', 'buildUrl', 'productIds', 'count', 'delete']));
+
+/**
+ * Builds the request URL.
+ *
+ * @param {Number|String} [id] Record ID
+ * @param {Object} [query] Query parameters
+ * @return {Object} URL object
+ * @private
+ */
+ProductListing.prototype.buildUrl = function(id, query) {
+  const baseUrl = base.buildUrl.call(this, id, query);
+
+  if (this.shopify.options.presentmentPrices) {
+    baseUrl.headers = { ['X-Shopify-Api-Features']: 'include-presentment-prices' };
```

Comment:
Style nit here and everywhere else: please try to keep the limit of 80 chars per line.